ush/ncl/field2d/fv3_Mslp_10m_Wind_plot.ncl

```diff
@@ -309,8 +311,8 @@ do ifil = startTimeLevel, endTimeLevel
    end if
 
    if (.not. isStormDomain) then
-      figName =  "./" + stormName + stormID + "." + tostring(startDate) + "." + modelName + ".mslp.10m_wind.f" + hour
```

Comment:
@KeqinWu-NOAA 
Could you please revert this figName related changes in all the ncl scripts. The figures need to be produced in the local directory. The driver scripts or workflow will take care of the copy or delivery of the figures to the right location.

Thanks!

Bin